02/Evgeny Vasiukevich/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ChangeEvent, useState } from 'react';
+import HotelsBlock from './components/HomesGuestsLoves/HotelsBlock';
+import SearchForm from './components/UI/SearchForm/SearchForm';
+import { hotelsData } from './data/hotelsData';
+
+function App() {
+  const [filterHotels, setFilterHotels] = useState(hotelsData);
+  const [renderHotels, setRenderHotels] = useState(filterHotels);
+
+  const handleFilter = (event: ChangeEvent<HTMLInputElement>): void => {
+    setFilterHotels(hotelsData.filter(hotel => 
+      (hotel.name + hotel.city + hotel.country).toUpperCase().includes(event.target.value.toUpperCase())
+    ));
+  }
+
+  const handleRender = (): void => setRenderHotels(filterHotels);
+
+  return (
+    <div className="App">
+      <HotelsBlock hotels={hotelsData} title='Homes guests loves'>{}</HotelsBlock>
```

Comment:
why do we need this empty block - {}

Reply:
In the third hw I removed it. Here it needs to add children in te second HotelsBlock, but in the first component it needs to avoid an error.